infrastructure/main.tf

```diff
@@ -17,12 +17,12 @@ module "db" {
   source             = "git@github.com:hmcts/cnp-module-postgres?ref=master"
   product            = var.product
   component          = var.component
-  name               = "rpe-${var.product}"
+  name               = "rpe-${var.product}-v11"
```

Comment:
you should be deploying a new db module not updating the old one this will cause the existing DB to be deleted and recreated